x-pack/test/functional/apps/infra/feature_controls/infrastructure_security.ts

```diff
@@ -63,7 +63,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       it('shows metrics navlink', async () => {
```

Comment:
Should we replace `metrics` here too?

```suggestion
      it('shows infrastructure navlink', async () => {
```